src/libstd/os/openbsd/raw.rs

```diff
@@ -25,6 +25,8 @@ use os::unix::raw::{uid_t, gid_t};
 #[stable(feature = "raw_ext", since = "1.1.0")] pub type off_t = i64;
 #[stable(feature = "raw_ext", since = "1.1.0")] pub type time_t = i64;
 
+#[unstable(feature = "pthread_t", issue = "0")] pub type pthread_t = usize;
```

Comment:
Could you connect all these issues to the same tracking issue as below? (#29791)